mav_trajectory_generation_ros/test/test_feasibility.cpp

```diff
@@ -0,0 +1,302 @@
+/*
+ * Copyright (c) 2016, Markus Achtelik, ASL, ETH Zurich, Switzerland
+ * Copyright (c) 2016, Michael Burri, ASL, ETH Zurich, Switzerland
+ * Copyright (c) 2016, Helen Oleynikova, ASL, ETH Zurich, Switzerland
+ * Copyright (c) 2016, Rik Bähnemann, ASL, ETH Zurich, Switzerland
+ * Copyright (c) 2016, Marija Popovic, ASL, ETH Zurich, Switzerland
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <random>
+
+#include <eigen-checks/gtest.h>
+
+#include <mav_trajectory_generation/polynomial_optimization_linear.h>
+#include <mav_trajectory_generation/segment.h>
+#include <mav_trajectory_generation/timing.h>
+#include <mav_trajectory_generation/vertex.h>
+
+#include "mav_trajectory_generation_ros/feasibility_recursive.h"
+#include "mav_trajectory_generation_ros/feasibility_sampling.h"
+#include "mav_trajectory_generation_ros/feasibility_analytic.h"
+
+using namespace mav_trajectory_generation;
+
+double createRandomDouble(double min, double max) {
```

Comment:
Can you stick this and maybe anything else you reuse into some kind of test utils? If it's only this ok to keep in 2 places, but if there's more...